recipes-bsp/boot0/boot0.bb

```diff
@@ -0,0 +1,39 @@
+DESCRIPTION = "Mainline friendly Secondary Program Loader for Allwinner D1"
+LICENSE = "GPL-2.0-or-later"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/GPL-2.0-or-later;md5=fed54355545ffd980b814dab4a3b312c"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+SRC_URI = " \
+	git://github.com/smaeul/sun20i_d1_spl;protocol=https;branch=mainline \
+	file://0001-config.mk-provide-path-to-linux-headers.patch \
+	file://0002-riscv-fix-build-with-binutils-2.38.patch \
+"
+
+PV = "1.0-dev"
```

Comment:
better to use `PV = "1.0+git${SRCPV}"` here

Reply:
I apply your suggestion.